encoding.bs

```diff
@@ -1241,6 +1314,88 @@ method, when invoked, must run these steps:
   </ol>
 </ol>
 
```

Comment:
Having an `<hr>` here would be good I think, to separate it a bit from the member definitions.